Tests/BasicPerformanceTests/SHA256PerfTests.swift

```diff
@@ -0,0 +1,31 @@
+/*
+ This source file is part of the Swift.org open source project
+ 
+ Copyright (c) 2014 - 2017 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+ 
+ See http://swift.org/LICENSE.txt for license information
+ See http://swift.org/CONTRIBUTORS.txt for Swift project authors
+ */
+
+import XCTest
+
+import Basic
+import TestSupport
+
+class SHA256PerfTests: XCTestCasePerf
+{
+    func test() {
+        measure {
+            let byte = "f"
+            let stream = BufferedOutputByteStream()
+            for _ in 0..<20000 {
+                stream <<< byte
```

Comment:
Since we're interested in performance of SHA256, leep the byte creation out of measure block.